projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -267,8 +511,21 @@ export function checkLinkedItemInViewport(index: number) {
  * Register GET quote route.
  */
 export function registerGetQuoteRoute(shopName: string) {
+  log(registerGetQuoteRoute.name);
   cy.intercept({
     method: 'GET',
     path: `${Cypress.env('OCC_PREFIX')}/${shopName}/users/current/quotes/*`,
   }).as(GET_QUOTE_ALIAS.substring(1)); // strip the '@'
 }
+/**
+ * Creates a Cypress log output
+ * @param functionName Name of the function that should be logged
+ * @param title Title of the test
+ */
+export function log(functionName: string, title?: string) {
+  if (title) {
+    cy.log(`########## ${title} // ${functionName} ##########`);
+  } else {
+    cy.log(`********** function: ${functionName} **********`);
+  }
+}
```

Comment:
parameter function name is used as log message, when title is provided as well. its a bit confusing i would split this method. As already stated before hand we can also fetch the title internally.

for the trace we could wrapp the function being executed, so you log start and end 
```suggestion
export function logWithTestTitle(message: string) {
  cy.log(`########## ${getTestTitle()} // ${message} ##########`);
}
export function trace(functionName: string, args?: Object, func?: Function) {
  if (args) {
    cy.log(
      `********** start function: ${functionName} with ${JSON.stringify(
        args
      )} **********`
    );
  } else {
    cy.log(`********** start function: ${functionName} **********`);
  }
  func();
  cy.log(`********** end function: ${functionName} **********`);
}
```

Reply:
example how to call the new trace
```
export function login(email: string, password: string, name: string): void {
  trace(login.name, arguments, () => {
    cy.get('cx-login [role="link"]')
      .click()
      .then(() => {
        cy.get('cx-login-form').should('be.visible');
      });
    authentication.login(email, password);
    cy.get('.cx-login-greet').should('contain', name);
    cy.get('cx-login').should('not.contain', 'Sign In');
  });
```

Reply:
created the new trace() method and moved it into the new logger class.